### PR TITLE
Iteration 3 tt

### PR DESCRIPTION
**Pull request by tylertomlinson**
Installed gems for color and text

Created OutputModule that includes:
TTY and Pastel
Moved messages hash from Game class 

Created Player class with methods:
player_placement
valid_placement
take_turn

Player test testing for:
Board object within Player class
Valid coordinates have been entered 
Board renders correctly after placing ships 
Player can take shot and opponents board reflects correctly

### CODE REVIEW

File: test/player_test.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  end
+
+  def test_player_can_place_ship_and_board_renders
+    @player1.valid_player_placement(@ship, @coordinates)
+    @player1.player_placement(@ship, @coordinates)
+
+    assert @player1.your_board.render(true).include?("S")
+  end
+
+  def test_player_can_take_turn
```
Comment (chrisstankus99) [inline test/player_test.rb:55]: Should we be testing that after the shot is taken the board renders correctly?

File: lib/output.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
+require 'pastel'
+require 'tty-font'
+
+module OutputModule
+
```
Comment (chrisstankus99) [inline lib/output.rb:5]: Can we move the cells instance variable into here like we did with messages?

File: test/player_test.rb (hunk 1 above)
Comment (tylertomlinson) [reply test/player_test.rb:55]: YES! We can maybe include that within our game test? Or we might already be testing that within the render method tests on cell.

File: lib/output.rb (hunk 2 above)
Comment (tylertomlinson) [reply lib/output.rb:5]: For sure. I'm down. Let's revisit once we have Iteration 3 functional. Can make the board dynamic and at the same time move to module.

**Pull request merged**